### PR TITLE
fix: Set the proper metric value for node allocatable metrics (HOTFIX)

### PR DESCRIPTION
**Pull request by bragi92**

### CODE REVIEW

File: source/plugins/ruby/KubernetesApiClient.rb [hunk 1]
```diff
@@ -517,7 +517,7 @@ def getContainerResourceRequestsAndLimits(pod, metricCategory, metricNameToColle
               #No container level limit for the given metric, so default to node level limit
             else
               if (metricCategory == "limits" && !nodeAllocatableRecord.nil? && !nodeAllocatableRecord.empty? && nodeAllocatableRecord.has_key?(metricNameToCollect))
-                metricValue = nodeAllocatableRecord[metricNameToCollect]
+                metricValue = getMetricNumericValue(metricNameToCollect, nodeAllocatableRecord[metricNameToCollect])
```
Comment (vishiy) [inline source/plugins/ruby/KubernetesApiClient.rb:520]: kaveesh, can you please search for places where this function is called (getMetricNumericValue) to see if there are any more occurences ?

Comment (ganga1980) [inline source/plugins/ruby/KubernetesApiClient.rb:520]: Thanks for investigating and making the fix. lets get the image tag update from "ciprod06272022" to "ciprod07282022" in omsagent.yaml and DockerFiles.

Comment (bragi92) [reply source/plugins/ruby/KubernetesApiClient.rb:520]: Found one other location, added it there.

**Review by ganga1980 — APPROVED** [review]
lgtm

**Pull request merged**